test/image/mocks/zzz-automargin-title-paper.json

```diff
@@ -0,0 +1,28 @@
+{
+    "data": [
+        {
+            "showlegend": false,
+            "type": "scatter",
+            "x": [
+            1,
+            2,
+            3
+            ],
+            "y": [
+            4,
+            5,
+            6
+            ]
+        }],
+    "layout": {
+        "height": 300,
+        "width": 400,
+        "margin": {"t":0, "b": 0, "l": 0, "r": 0},
+        "title": {
+            "automargin": true, 
+            "text": "Paper | no-pad | y=1",
```

Comment:
Would you please add yet another mock with multi-line text (either 2 or 3 lines) and commit the baseline, even if it may not position perfectly? This would help use debug and possibly fix the issue.
Thank you!